frontend/src/component/admin/apiToken/ProjectsList/ProjectsList.tsx

```diff
@@ -84,5 +80,5 @@ export const ProjectsList: FC<IProjectsListProps> = ({ projects, project }) => {
         return <LinkCell to={`/projects/${item}`} title={item} />;
     }
 
-    return null;
+    return '';
```

Comment:
why are we changing the return to an empty string here? wouldn't it be better to return `null`?

Reply:
Does not matter much. This looked more consistent with tests

Reply:
Right. I don't really know if it matters, but the [docs use null](https://react.dev/learn/conditional-rendering#conditionally-returning-nothing-with-null). Happy for you to keep this one, though. What do we do in other components?